packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -258,19 +258,21 @@ const TeamPromptResponseCard = (props: Props) => {
               readOnly={!isCurrentViewer}
               placeholder={'Share your response...'}
             />
-            <ResponseCardFooter>
-              <StyledReactjis reactjis={reactjis || []} onToggle={onToggleReactji} />
-              <ReplyButton onClick={() => onSelectDiscussion()}>
-                {replyCount > 0 ? (
-                  <>
-                    <TeamPromptRepliesAvatarList edgesRef={discussionEdges} />
-                    {replyCount} {plural(replyCount, 'Reply', 'Replies')}
-                  </>
-                ) : (
-                  'Reply'
-                )}
-              </ReplyButton>
-            </ResponseCardFooter>
+            {!!contentJSON && (
```

Comment:
+1 to make it easier to read and understand, can we declare a variable for this? `hasResponse`? also, it'd be better to check `plaintextContent` as populated `contentJSON` still can have an empty response